content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -0,0 +1,285 @@
+
+1. Set the default version for all services to v1.
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/networking/virtual-service-all-v1.yaml@
+    {{< /text >}}
+
+## Rate limits
+
+In this task, you configure Istio to rate limit traffic to `productpage` whenever request is routed to a specific path.
+Rate limit using Envoy can be achieved using either Global Rate Limiting or Local Rate Limiting.
+
+### Global Rate Limit
+
+Envoy can be used to setup global rate limit for your mesh. More info on it can be found [here](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_features/global_rate_limiting)
+
+1.  For Global Rate Limit, you need a global gRPC rate limiting service. You can either use Envoy's [reference implementation](https://github.com/envoyproxy/ratelimit) written in Go which uses a Redis backend or implement a service that implements the Envoy's defined RPC/IDL protocol.
+    In the definition of the service you also define the rate limits and the descriptor on which you want
+    to rate limit on. For example, below is the configmap defining rate limit on PATH header.
+
+    {{<text yaml >}}
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: ratelimit-config
+    data:
+      config.yaml: |
+        domain: echo-ratelimit
+        descriptors:
+          - key: PATH
+            value: "/productpage"
+            rate_limit:
+              unit: minute
+              requests_per_unit: 1
+          - key: PATH
+            rate_limit:
+              unit: minute
+              requests_per_unit: 100
+
+    {{< /text >}}
+
+1.  Apply EnvoyFilter Patch in Istio that enables global rate limit. The below patch applies rate limiting
+    on any traffic through  Istio Ingressgateway and  matches  it on the  Path header of the request.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: EnvoyFilter
+    metadata:
+      name: filter-ratelimit
+      namespace: istio-system
+    spec:
+      workloadSelector:
+        # select by label in the same namespace
+        labels:
+          istio: ingressgateway
+      configPatches:
+        # The Envoy config you want to modify
+        - applyTo: HTTP_FILTER
+          match:
+            context: GATEWAY
+            listener:
+              filterChain:
+                filter:
+                  name: "envoy.http_connection_manager"
+                  subFilter:
+                    name: "envoy.router"
+          patch:
+            operation: INSERT_BEFORE
+            # Adds the Envoy Rate Limit Filter in HTTP filter chain.
+            value:
+              name: envoy.ratelimit
+              typed_config:
+                "@type": type.googleapis.com/envoy.config.filter.http.rate_limit.v2.RateLimit
+                # domain can be anything! Match it to the ratelimter service config
+                domain: productpage-ratelimit
```

Comment:
above, we used `echo-ratelimit`. should we aim to be consistent in our examples?

Reply:
Thanks for pointing